tuist/tuist
# Fix implicit-imports command not reporting implicit imports of external dependencies

Resolves https://github.com/tuist/tuist/issues/6697

### Short description 📝

Added support for testing external dependencies on implicitness. Now instead of using allInternalTargets names as a mask for imports, using allTargets names.  

### How to test the changes locally 🧐

```
git clone https://github.com/karwa/tuist-external-implicit-dependency
tuist inspect implicit-imports tuist-external-implicit-dependency // Somewhere in xcode 

```
> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/get-started) for reference).

### Contributor checklist ✅


### Reviewer checklist ✅
